extensions/amp-pinterest/0.1/pin-widget.js

```diff
@@ -0,0 +1,194 @@
+import {assertHttpsUrl} from '../../../src/url';
+
+import {Util} from './util';
+
+const makePin = () => {
+
+  // make an embedded pin widget
+  // pinId will be inferred from pinUrl
+
+  let pinId = '';
+  try {
+    pinId = pinUrl.split('/pin/')[1].split('/')[0];
+  } catch (err) {
+    // fail silently
+    return;
+  }
+
+  const width = this.element.getAttribute('data-width');
+
+  const structure = make({'span': {}});
+
+  const renderPin = function(r) {
```

Comment:
do we need to switch `renderPin` into an arrow function? i believe theres a couple of references to `this` inside of it and i dont see us doing any `bind` calls (could be wrong). i believe this was the issue being fixed in https://github.com/ampproject/amphtml/pull/1062